setup.py

```diff
@@ -122,6 +122,9 @@ def _format_pulp_requirement(plugin, specifier=None, ref=None, gh_namespace="pul
     "social-auth-app-django>=3.1.0,<4.0.0",
     "dynaconf>=3.1.12",
     "django-auth-ldap==4.0.0",
+    "distro",
+    "boto3",
```

Comment:
@dmzoneill turns out we already have these 2 dependencies in our galaxy-ng env used for beta-galaxy... these are pulled in via pulp and the way we use storages. You can remove them from here.

```
(venv) sh-4.4$ pip freeze | fgrep distro
distro==1.8.0
(venv) sh-4.4$ pip freeze | fgrep boto3
boto3==1.26.104
```